.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarks Test
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  Benchmarks:
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      matrix:
+        runner: [ubuntu-latest, self-hosted]
+        benchmark: [sysbench, getpid]
+      fail-fast: false
+    timeout-minutes: 60
+    container: 
+      image: asterinas/asterinas:0.5.0
+      options: --device=/dev/kvm
+    env:
+      RUSTUP_DIST_SERVER: https://mirrors.ustc.edu.cn/rust-static
+      RUSTUP_UPDATE_ROOT: https://mirrors.ustc.edu.cn/rust-static/rustup
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up the environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y jq
+        chmod +x regression/benchmark/benchmarks.sh
+        git config --global --add safe.directory /__w/asterinas/asterinas
+        git config --global http.sslVerify false
+        git config --global http.version HTTP/1.1
```

Comment:
What is the purpose of these unconventional setup? Better explain the surprising code with comments.